lib/redis_client.rb

```diff
@@ -431,7 +431,7 @@ def pipelined
       results = ensure_connected(retryable: pipeline._retryable?) do |connection|
         commands = pipeline._commands
         @middlewares.call_pipelined(commands, config) do
-          connection.call_pipelined(commands, pipeline._timeouts)
+          connection.call_pipelined(commands, pipeline._timeouts, exception: pipeline._raise_exception)
```

Comment:
Hum, in the end, the only place where we use it is here, so we could just forward the `pipelined` argument rather than to store this on the `Pipeline` object, or am I missing something?

Reply:
You're absolutely right, this was part of a quick test I did for my upcoming change in `redis-rb`. Let me revert this atrocity.